### PR TITLE
Add oc create Commands that Reference YAML Files Under Resources

### PR DESCRIPTION
**Pull request by danielhelfand**
This pull request adds the following `oc create` commands to the main `README` to allow users to be able to copy and paste the commands to create the `pipeline`, `pipelineresources`, and trigger the `pipeline` via a `pipelinerun`. The commands added reference the YAML files under the resources folder of this repository. 

Additionally, the `tkn` download path has been updated to https://github.com/tektoncd/cli#getting-started to give more explicit instructions on downloading the CLI.

Finally, there were some minor grammar and writing tweaks made.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -200,7 +206,7 @@ Check the list of pipelines you have created using the CLI:
 ```
 $ tkn pipeline ls
 NAME             AGE              LAST RUN   STARTED   DURATION   STATUS
-build-pipeline   25 seconds ago   ---        ---       ---        ---
+deploy-pipeline  25 seconds ago   ---        ---       ---        ---
```
Comment (danielhelfand) [inline README.md:209]: This was changed since the `pipeline` used in the tutorial is actually `deploy-pipeline`

**Comment by danielhelfand** [comment]
@siamaksade @vdemeester Whenever either of you has a moment, could you review this? Thanks!

**Review by siamaksade — APPROVED** [review]

**Comment by danielhelfand** [comment]
@siamaksade Appreciate it!

**Pull request merged**